### PR TITLE
*: fix several typos

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Fix several typos

Ref: https://github.com/tikv/tikv/pull/16931

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/17405
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-monitoring-and-alert.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ aliases: ['/zh/tidb/dev/pitr-monitoring-and-alert/']
 
 | 指标                                                    | 类型        | 说明                                                                                                                                              |
 |-------------------------------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------|
-| **tikv_log_backup_interal_actor_acting_duration_sec** | Histogram | 处理内部各种消息事件的耗时。<br/>`message :: TaskType`                                                                                                        |
+| **tikv_log_backup_internal_actor_acting_duration_sec** | Histogram | 处理内部各种消息事件的耗时。<br/>`message :: TaskType`                                                                                                        |
```
Comment (lilin90) [inline br/br-monitoring-and-alert.md:31]: To be confirmed.

File: releases/release-rc.1.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: TiDB RC1 Release Notes
 aliases: ['/docs-cn/dev/releases/release-rc.1/','/docs-cn/dev/releases/rc1/']
-summary: TiDB RC1 于 2016 年 12 月 23 日发布，TiKV 提升了写入速度和稳定性，支持百 TB 级别数据，集群规模支持 200 个节点。PD 优化了调度策略框架，添加了 label 支持，提供了 PD Controler。TiDB 新增了 SQL 查询优化器和更多 MySQL 内建函数，重构了 time 相关类型的实现，提升了和 MySQL 的兼容性。工具方面，Loader 兼容 Percona 的 Mydumper 数据格式，提供了多线程导入、出错重试、断点续传等功能，并且针对 TiDB 有优化。完成了一键部署工具。
+summary: TiDB RC1 于 2016 年 12 月 23 日发布，TiKV 提升了写入速度和稳定性，支持百 TB 级别数据，集群规模支持 200 个节点。PD 优化了调度策略框架，添加了 label 支持，提供了 PD Controller。TiDB 新增了 SQL 查询优化器和更多 MySQL 内建函数，重构了 time 相关类型的实现，提升了和 MySQL 的兼容性。工具方面，Loader 兼容 Percona 的 Mydumper 数据格式，提供了多线程导入、出错重试、断点续传等功能，并且针对 TiDB 有优化。完成了一键部署工具。
```
Comment (Oreoxmt) [inline releases/release-rc.1.md:4]:
```suggestion
summary: TiDB RC1 于 2016 年 12 月 23 日发布，TiKV 提升了写入速度和稳定性，支持百 TB 级别数据，集群规模支持 200 个节点。PD 优化了调度策略框架，添加了 label 支持，提供了 PD Control。TiDB 新增了 SQL 查询优化器和更多 MySQL 内建函数，重构了 time 相关类型的实现，提升了和 MySQL 的兼容性。工具方面，Loader 兼容 Percona 的 Mydumper 数据格式，提供了多线程导入、出错重试、断点续传等功能，并且针对 TiDB 有优化。完成了一键部署工具。
```

File: releases/release-rc.1.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ summary: TiDB RC1 于 2016 年 12 月 23 日发布，TiKV 提升了写入速度
 
 + PD 调度策略框架优化，策略更加灵活合理
 + 添加 label 支持，支持跨 DC 调度
-+ 提供 PD Controler，方便操作 PD 集群
++ 提供 PD Controller，方便操作 PD 集群
```
Comment (Oreoxmt) [inline releases/release-rc.1.md:23]:
```suggestion
+ 提供 PD Control，方便操作 PD 集群
```

File: br/br-monitoring-and-alert.md (hunk 1 above)
Comment (lilin90) [inline br/br-monitoring-and-alert.md:31]: The code has been fixed in https://github.com/tikv/tikv/pull/16931.

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**